### ISSUE TITLE
Unexpected token <

### ISSUE THREAD

**Issue by Yuripetusko** [makotot/react-scrollspy#1]
I get this error when trying to load this module
```
Module parse failed: /Users/***/Sites/***/node_modules/react-scrollspy/src/js/lib/scrollspy.jsx Line 110: Unexpected token <
```

**Comment by Yuripetusko** [comment]
```
ERROR in ./~/react-scrollspy/src/js/lib/scrollspy.jsx
Module parse failed: /Users/***/Sites/***/node_modules/react-scrollspy/src/js/lib/scrollspy.jsx Line 110: Unexpected token <
You may need an appropriate loader to handle this file type.
| 
|     return (
|       <nav>
|         <ul className={ this.props.className ? this.props.className : '' }>
|           { items }
```

**Comment by makotot** [comment]
@Yuripetusko 
If you are using webpack, you need to configure module loaders in your webpack.config like this.
```js
test: /\.jsx?$/,
```
[You need to transform the file that end with `jsx` extension.](http://stackoverflow.com/a/31374937)

**Comment by Yuripetusko** [comment]
Thanks @makotot for a quikc reply. In StackOverflow answer you posted it is actually said
```
but the best solution is that packages not use JSX in the published version of the JavaScript—
```
And link to this PR https://github.com/thomasboyt/react-spin/pull/3/files

maybe you can do something like that here?

**Comment by makotot** [comment]
@Yuripetusko 
It fixed in v0.1.3.
